src/uu/more/src/more.rs

```diff
@@ -274,18 +280,33 @@ fn more(
     buff: &str,
     stdout: &mut Stdout,
     next_file: Option<&str>,
-    options: &Options,
+    options: &mut Options,
 ) -> UResult<()> {
     let (cols, mut rows) = terminal::size().unwrap();
     if let Some(number) = options.lines {
         rows = number;
     }
 
-    let lines = break_buff(buff, usize::from(cols));
+    let mut lines = break_buff(buff, usize::from(cols));
```

Comment:
according to the coverage, the test doesn't go here, would it be possible to test that? thanks

Reply:
I have added a test for it, but I believe the coverage doesn't detect it because the tests for `more` are all contained within an `if std::io::stdout().is_terminal()` statement.